### PR TITLE
Add Perks Tutorial Part 2

### PR DESCRIPTION
**Pull request by CommanderFoo**
Not sure if we are sticking with the name "Perk Systems", for now I have appended "Part 2" until we think of something better.

### CODE REVIEW

File: src/tutorials/perks_tutorial_part2.md [hunk 1]
```diff
@@ -0,0 +1,1609 @@
...
+        if not hasVIP and playerData.vip == 1 then
+            playerData.vip = 0
+            player:SetResource("showvipnotice", 1)
+            Storage.SetPlayerData(player, playerData)
+        end
+    end
+end
+```
+
+#### The VIPServerScript
```
Comment (DoubleA-MG) [inline src/tutorials/perks_tutorial_part2.md]: #### The VIPShopServer Script

File: src/tutorials/perks_tutorial_part2.md [hunk 2]
```diff
@@ -0,0 +1,1609 @@
...
+
+??? "VIPServer"
```
Comment (DoubleA-MG) [inline src/tutorials/perks_tutorial_part2.md]: ??? "VIPShopServer"

File: src/tutorials/perks_tutorial_part2.md [hunk 3]
```diff
@@ -0,0 +1,1609 @@
...
+                    end
+                end
+            else
+                status.senderMessage = "Command data \"" .. item .. "\" does not exist."
+            end
+        end
+
+    })
+    ```
+
```
Comment (DoubleA-MG) [inline src/tutorials/perks_tutorial_part2.md:1179]: perhaps a warning to update admins if they copy and pasted the whole file (like me)

File: src/tutorials/perks_tutorial_part2.md [hunk 4]
```diff
@@ -0,0 +1,1609 @@
...
```
Comment (DoubleA-MG) [inline src/tutorials/perks_tutorial_part2.md:1581]: perhaps a warning to update admins if they copy and pasted the whole file (like me)

**Review by DoubleA-MG — APPROVED** [review]
Looks great to me!

**Comment by StanzillaManticore** [comment]
@PopThosePringles Wanna add the forums link now? Then I'll include it in today's deploy.

**Pull request merged**